src/go/rpk/go.mod

```diff
@@ -20,6 +20,7 @@ require (
 	github.com/google/uuid v1.1.1
 	github.com/hashicorp/go-multierror v1.1.0
 	github.com/icza/dyno v0.0.0-20200205103839-49cb13720835
+	github.com/imdario/mergo v0.3.12
```

Comment:
This is no longer needed, please run `go mod tidy`

Reply:
Hmm, and yet, the build failed saying this is needed.

Reply:
Oh, need to run `go mod tidy` in `../k8s`

Reply:
Done!